### PR TITLE
Add 13 new faculties for Northeastern University

### PR DESCRIPTION
**Pull request by shawnricecake**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by shawnricecake** [comment]
These qualified faculties are recruited during two years (2020 and 2021) \
and they are selected from the official lists given by the CS department of Northeastern University: \
[ 2021 new faculties ](https://www.khoury.northeastern.edu/khoury-welcomes-new-faculty-for-2021-22/)  \
[ 2020 new faculties ](https://www.khoury.northeastern.edu/press_releases/khoury-welcomes-new-faculty-hires-for-2020-21/)

File: csrankings-m.csv [hunk 1]
```diff
@@ -1752,6 +1752,7 @@ Michael Zbyszynski,Goldsmiths University of London,https://www.gold.ac.uk/comput
 Michael Zink,University of Massachusetts Amherst,http://www.ecs.umass.edu/ece/zink,iVck6WIAAAAJ
 Michail Matthaiou,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/Staff/BusinessCard/?name=m.matthaiou,g6HXpqoAAAAJ
 Michail N. Giannakos,NTNU,https://www.ntnu.edu/employees/michailg,USOZIeMAAAAJ
+Michal Aibin,Northeastern University,https://www.khoury.northeastern.edu/people/michal-aibin/,lAJCBiQAAAAJ
```
Comment (emeryberger) [inline csrankings-m.csv]: Not at the main campus - please remove.

File: csrankings-r.csv [hunk 2]
```diff
@@ -134,6 +134,7 @@ Rahul Shah,Louisiana State University,http://www.csc.lsu.edu/~rahul,ZpkdT2UAAAAJ
 Rahul Sharma 0003,University of Queensland,http://researchers.uq.edu.au/researcher/2660,UhDW6jkAAAAJ
 Rahul Simha,George Washington University,https://www.seas.gwu.edu/~simha,NOSCHOLARPAGE
 Rahul Vaze,Tata Inst. of Fundamental Research,http://www.tcs.tifr.res.in/~vaze,NOSCHOLARPAGE
+Raimond L. Winslow,Northeastern University,https://roux.northeastern.edu/people/raimond-winslow/,xc3Z9lMAAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv]: Not at the main campus. Please remove.

File: csrankings-r.csv [hunk 3]
```diff
@@ -649,6 +650,7 @@ Richard Han,University of Colorado Boulder,http://www.cs.colorado.edu/~rhan,mah-
 Richard Harper 0001,Lancaster University,https://www.lancaster.ac.uk/scc/about-us/people/richard-harper,RGVGixkAAAAJ
 Richard Hartley 0001,Australian National University,http://axiom.anu.edu.au/~hartley,cHia5p0AAAAJ
 Richard Haverkamp,Massey University,http://www.massey.ac.nz/massey/expertise/profile.cfm?stref=627430,UanjyZMAAAAJ
+Richard Hoshino,Northeastern University,https://richardhoshino.com/,a9DxsY4AAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv]: Not at the main campus. Please remove.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-m.csv (hunk 1 above)
Comment (shawnricecake) [reply csrankings-m.csv]: I have deleted that person

File: csrankings-r.csv (hunk 3 above)
Comment (shen494157765) [reply csrankings-r.csv:652]: I have deleted Raimond L. Winslow

File: csrankings-r.csv (hunk 2 above)
Comment (shawnricecake) [reply csrankings-r.csv]: I have deleted Raimond L. Winslow

File: csrankings-r.csv (hunk 3 above)
Comment (shawnricecake) [reply csrankings-r.csv]: I have deleted Richard Hoshino

**Comment by shawnricecake** [comment]
I have finished the removal of three faculties who are not at the main campus of NEU.
Thanks

**Review by emeryberger — APPROVED** [review]

**Pull request merged**